var/spack/repos/builtin/packages/meme/package.py

```diff
@@ -35,9 +35,13 @@ class Meme(AutotoolsPackage):
 
     version('4.11.4', '371f513f82fa0888205748e333003897')
 
+    variant('mpi', default=True, description='Enable MPI support')
+
     depends_on('zlib', type=('link'))
     depends_on('libxml2', type=('link'))
     depends_on('libxslt', type=('link'))
     depends_on('libgcrypt', type=('link'))
     depends_on('perl', type=('build', 'run'))
     depends_on('python@2.7:', type=('build', 'run'))
+    # openmpi support
+    depends_on('openmpi@2:', type=('link'))
```

Comment:
spack install "meme -mpi" and spack install meme ~mpi now work using:

```
depends_on('openmpi@2:', when='+mpi', type=('link'))

    # this will disable mpi
    def configure_args(self):
        spec = self.spec
        args = []
        if '-mpi' in spec:
            args += ['--enable-serial']
        if '~mpi' in spec:
            args += ['--enable-serial']
        return args
```


Though "spack install meme -mpi" produces:

```
usage: spack [-h] [-d] [-D] [-k] [-m] [-p] [-P STAT] [--lines LINES] [-v] [-s]
             [-V]
             [COMMAND]
spack: error: argument -p/--profile: ignored explicit argument 'i'
```

So do you prefer we use 

> ~

?